tests/scripts/mbedtls_test.py

```diff
@@ -79,8 +79,7 @@ def __escaped_split(inp_str, split_char):
         split_colon_fn = lambda x: re.sub(r'\\' + split_char, split_char, x)
         if len(split_char) > 1:
             raise ValueError('Expected split character. Found string!')
-        out = map(split_colon_fn, re.split(r'(?<!\\)' + split_char, inp_str))
-        out = [x for x in out if x]
+        out = list(map(split_colon_fn, re.split(r'(?<!\\)' + split_char, inp_str)))
```

Comment:
The code used to remove empty pieces, but no longer does. I don't understand why empty pieces would have been removed, so it may have been a bug, but since you're changing the behavior, please explain why in the commit message.

Reply:
I think it's related to https://github.com/ARMmbed/mbedtls/pull/2207/files#r234792082
In addition, I used the script from https://github.com/ARMmbed/mbedtls/issues/2653#issuecomment-496234452 as reference and modified it to fit the correct versions.
I'll return this line